e2e/e2e_test.go

```diff
@@ -23,7 +23,7 @@ func init() {
 	flag.BoolVar(&testCephFS, "test-cephfs", true, "test cephfs csi driver")
 	flag.BoolVar(&testRBD, "test-rbd", true, "test rbd csi driver")
 	flag.BoolVar(&upgradeTesting, "upgrade-testing", false, "perform upgrade testing")
-	flag.StringVar(&upgradeVersion, "upgrade-version", "v2.1.2", "target version for upgrade testing")
+	flag.StringVar(&upgradeVersion, "upgrade-version", "v3.3.1", "target version for upgrade testing")
```

Comment:
this should be a separate commit

Reply:
https://github.com/ceph/ceph-csi/blob/798437d0c4cc970187f65ec110684b597f98cf49/e2e/README.md#L96
This line also requires the same change.

Reply:
The idea here was to get the confirmation that tests are passing. As we have it now, updating the PR with the required changes.